src/frontend/admin/src/services/api/models/OrderGroup.ts

```diff
@@ -1,11 +1,13 @@
 import { CourseProductRelation } from "@/services/api/models/Relations";
+import { Discount } from "@/services/api/models/Discount";
 
 export type OrderGroup = {
   id: string;
   nb_available_seats: number;
```

Comment:
With api changes this type should be update.
```suggestion
  nb_available_seats: number | null;
```